extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -142,7 +142,7 @@ export class BaseCarousel extends AMP.BaseElement {
   }
 
   /** @override */
-  documentInactiveCallback() {
+  unlayoutCallback() {
```

Comment:
`PAUSED` shouldn't cause anything to unload (throw away `Window`s, etc) since that would affect performance and look really strange during the swipe.

How about detecting we detect if the the element is in the viewport inside `scheduleUnload`, and skip it if it is?